### PR TITLE
Add localisation to voice wizard

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [X] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-check.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export class HaVoiceAssistantSetupStepCheck extends LitElement {
               src="/static/images/voice-assistant/hi.png"
               alt="Casita Home Assistant hi logo"
             />
-            <h1>Hi</h1>
+            <h1>Hello</h1>
             <p class="secondary">
               Over the next couple steps we're going to personalize your voice
```
Comment (silamon) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-check.ts:65]: One more.

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts [hunk 2]
```diff
@@ -69,53 +69,89 @@ export class HaVoiceAssistantSetupStepPipeline extends LitElement {
...
           <div class="message user ${this._showFirst ? "show" : ""}">
             ${!this._showFirst ? "…" : "Turn on the lights in the bedroom"}
           </div>
           ${this._showFirst
-            ? html`<div class="timing user">0.2 seconds</div>`
+            ? html`<div class="timing user">
+                0.2
+                ${this.hass.localize(
+                  "ui.panel.config.voice_assistants.satellite_wizard.pipeline.seconds"
+                )}
```
Comment (silamon) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts:92]: We have a new duration formatting available that may be more suitable for all these.

Comment (bramkragten) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts:92]: Problem is that it only supports integers and not float values :-/

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts [hunk 3]
```diff
@@ -69,53 +69,89 @@ export class HaVoiceAssistantSetupStepPipeline extends LitElement {
...
-        How quickly your assistant responds depends on the power of the
-        hardware.
+        ${this.hass.localize(
+          "ui.panel.config.voice_assistants.satellite_wizard.pipeline.secondary"
+        )}
       </p>
       <div class="container">
         <div class="messages-container cloud">
           <div class="message user ${this._showFirst ? "show" : ""}">
             ${!this._showFirst ? "…" : "Turn on the lights in the bedroom"}
```
Comment (MindFreeze) [inline src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts:85]: These would be a bit weird when everything else is in a different language but translating them could make things worse :/

**Review by MindFreeze — APPROVED** [review]

File: src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts (hunk 3 above)
Comment (bramkragten) [reply src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts:85]: Yeah, I want to look into translating them, but the length of the text is kinda important

File: src/translations/en.json [hunk 4]
```diff
@@ -2825,6 +2829,83 @@
...
+              "no_selection": "Please select an area"
+            },
+            "pipeline": {
+              "title": "What hardware do you want to use?",
+              "secondary": "How quickly your assistant responds depends on the power of the hardware.",
+              "seconds": "seconds",
+              "cloud": {
+                "description": "Ideal if you don't have a powerful system at home."
+              },
+              "local": { "title": "", "description": "", "setup": "" }
```
Comment (balloob) [inline src/translations/en.json:2875]: @bramkragten looks like you missed this info

**Pull request merged**